pom.xml

```diff
@@ -61,4 +61,29 @@
         <module>tests</module>
         <module>mule-wrapper-additional-parameters-parser</module>
     </modules>
+
+    <profiles>
+        <profile>
+            <id>testJdkWithoutModulePath</id>
```

Comment:
This way we can prevent tests with Java 11 to run in the module-path, which we actually want (and **need** in this case) only for Java 17.